src/include/miopen/serializable.hpp

```diff
@@ -82,24 +82,23 @@ struct Serializable
             ok = Parse<T>::apply(part, x);
         }
     };
+
+    // cppcheck-suppress constParameter
```

Comment:
I think it would be better to keep the bind instead of using `cppcheck-suppress`.